doc/secrets.md

```diff
@@ -15,3 +15,4 @@ Currently used keys are listed in `SecretKey.values`. For example:
 - SMTP account username: `smtp.username`
 - SMTP account password: `smtp.password`
 - Redis connection string: `redis.connectionString`
+- OAuth `secret_key`: `oauth.secret-[client_id]`
```

Comment:
This is the site oauth flow... please note that and refer to the `oauth.md` file.